src/main.c

```diff
@@ -104,56 +104,56 @@ static struct module_data self = {
 	.msg_q = &msgq_app,
 };
 
-static char *state2str(enum app_state state)
+static char *state2str(enum state_type state)
 {
 	switch (state) {
-	case APP_STATE_INIT:
-		return "APP_STATE_INIT";
-	case APP_STATE_RUNNING:
-		return "APP_STATE_RUNNING";
+	case STATE_INIT:
+		return "STATE_INIT";
+	case STATE_RUNNING:
+		return "STATE_RUNNING";
 	default:
 		return "Unknown";
 	}
 }
 
-static char *sub_state2str(enum app_sub_state state)
+static char *sub_state2str(enum sub_state_type state)
```

Comment:
`state` -> `new_state` to avoid confusion with global variable `state`

Reply:
Agreed. Updated the other modules as well.